content/en/blog/_posts/2018-12-05-kubeadm-ga-release.md

```diff
@@ -14,7 +14,7 @@ The goal of kubeadm is to provide a foundational implementation for bootstrappin
 
 ### The scope of kubeadm
 
-kubeadm is focused on bootstrapping Kubernetes clusters on existing infrastructure and performing an essential set of maintenance tasks. The core of the kubeadm interface is quite simple: new control plane nodes are created by running `kubeadm init` and worker nodes are joined to the control plane by running `kubeadm join`. Also included are utilities for managing already bootstrapped clusters, such as control plane upgrades and token and certificate renewal.
+kubeadm is focused on bootstrapping Kubernetes clusters on existing infrastructure and performing an essential set of maintenance tasks. The core of the kubeadm interface is quite simple: new control plane nodes are created by running [`kubeadm init`](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/) and worker nodes are joined to the control plane by running [`kubeadm join`](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/). Also included are utilities for managing already bootstrapped clusters, such as control plane upgrades and token and certificate renewal.
```

Comment:
nit: capitalize Kubeadm when starting a sentence?

Reply:
This is not currently done in the kubeadm documentation (for example [here](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/)). I'm following the existing usage but am 100% happy to defer to you on that.

Reply:
lowercase kubeadm at the start of a sentence is fine.